.github/workflows/test-lint-release.yml

```diff
@@ -112,19 +115,53 @@ jobs:
         env:
           CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
 
+  Docker:
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    needs:
+      - release
+    strategy:
+      matrix:
```

Comment:
based on [this](https://stackoverflow.com/questions/66337210/is-it-possible-to-push-docker-images-for-different-architectures-separately), it is very likely that having two different push steps pushing to the same image does not work